libusb1-sys/src/lib.rs

```diff
@@ -204,6 +204,88 @@ pub struct libusb_pollfd {
     pub events: c_short,
 }
 
+#[repr(C)]
+pub union libusb_init_option__value {
+    pub ival: c_int,
+    pub log_cbval: libusb_log_cb,
+}
+
+#[repr(C)]
+pub enum libusb_option {
+    /** Set the log message verbosity.
```

Comment:
If you begin copying documentation, I suggest converting it to Rust doc format.

Reply:
Addressed, and squashed into previous commit, thanks!